mdx_linkify/mdx_linkify.py

```diff
@@ -5,23 +5,31 @@
 
 
 class LinkifyPostprocessor(Postprocessor):
-    def __init__(self, md, linkify_callbacks=[]):
+    def __init__(self, md, linkify_callbacks=[], linkify_parse_email=False):
         super(Postprocessor, self).__init__(md)
         self._callbacks = linkify_callbacks
+        self._parse_email = linkify_parse_email
 
     def run(self, text):
         text = bleach.linkify(text,
+                              parse_email=self._parse_email,
                               callbacks=self._callbacks,
                               skip_tags=['code'])
         return text
 
 
 class LinkifyExtension(Extension):
-    config = {'linkify_callbacks': [[], 'Callbacks to send to bleach.linkify']}
+    
+    def __init__(self, **kwargs):
+        self.config = {
+            'linkify_callbacks': [[], 'Callbacks to send to bleach.linkify'],
+            'linkify_parse_email': [False, 'Parse email addresses'],
+        }
+        super(LinkifyExtension, self).__init__(**kwargs)
```

Comment:
This way the configuration parameters can be over-ridden at run time (as a test I added shows): https://python-markdown.github.io/extensions/api/#configsettings